internal/declcfg/load.go

```diff
@@ -71,32 +71,32 @@ func extractCSV(objs []string) string {
 	return ""
 }
 
-func readJSON(r io.Reader) (*DeclarativeConfig, error) {
+func readYAMLOrJSON(r io.Reader) (*DeclarativeConfig, error) {
 	cfg := &DeclarativeConfig{}
-	dec := json.NewDecoder(r)
-	for dec.More() {
-		doc := &json.RawMessage{}
-		if err := dec.Decode(doc); err != nil {
+	dec := yaml.NewYAMLOrJSONDecoder(r, 16)
```

Comment:
Any specific reason we're using a buffer size of 16 here, instead of something like 100?

Reply:
I'll set this to 4096. That looks like a typical value used in `kubernetes/kubernetes`.

```suggestion
	dec := yaml.NewYAMLOrJSONDecoder(r, 4096)
```